web/themes/custom/liibarrio2020/css/main.css

```diff
@@ -10749,4 +10726,17 @@ body.gazettes-search-year .gazette-date-facet-list {
        -o-animation: fadeout 2s;
           animation: fadeout 2s; }
 
+.toc-container {
```

Comment:
This is a compiled file, changes must be made https://github.com/africanlii/liiweb/blob/master/web/themes/custom/liibarrio2020/scss/components/_table-of-contents.scss instead and then compiled. See https://github.com/africanlii/liiweb#changing-css-and-js-in-the-liibarrio2020-theme